ts/features/pn/aar/utils/stateUtils.ts

```diff
@@ -52,6 +53,73 @@ type FinalNotAddressee = {
   iun: string;
 };
 
+type NotAddressee = {
+  type: SendAARFlowStatesType["notAddressee"];
+  recipientInfo: RecipientInfo;
+  qrCode: string;
+  iun: string;
+};
+
+type CreateMandate = {
+  type: SendAARFlowStatesType["creatingMandate"];
+  recipientInfo: RecipientInfo;
+  qrCode: string;
+  iun: string;
+};
+
+type CieCanAdvisory = {
+  type: SendAARFlowStatesType["cieCanAdvisory"];
+  recipientInfo: RecipientInfo;
+  iun: string;
+  mandateId: string;
+  verificationCode: string;
+};
+
+type CieCanInsertion = {
+  type: SendAARFlowStatesType["cieCanInsertion"];
+  recipientInfo: RecipientInfo;
+  iun: string;
+  mandateId: string;
+  verificationCode: string;
+};
+
+type CieScanningAdvisory = {
+  type: SendAARFlowStatesType["cieScanningAdvisory"];
+  recipientInfo: RecipientInfo;
+  iun: string;
+  mandateId: string;
+  verificationCode: string;
+  can: string;
+};
+
+type AndroidNFCActivation = {
+  type: SendAARFlowStatesType["androidNFCActivation"];
+  recipientInfo: RecipientInfo;
+  iun: string;
+  mandateId: string;
+  verificationCode: string;
+  can: string;
+};
+
+type CieScanning = {
+  type: SendAARFlowStatesType["cieScanning"];
+  recipientInfo: RecipientInfo;
+  iun: string;
+  mandateId: string;
+  verificationCode: string;
+  can: string;
+};
+
+type ValidateMandate = {
+  type: SendAARFlowStatesType["validatingMandate"];
+  recipientInfo: RecipientInfo;
+  iun: string;
+  mandateId: string;
+  mrtdData: InternalAuthAndMrtdResponse["mrtd_data"];
```

Comment:
What about not having a direct dependency to the SDK and declaring this data structure using native types (i.e., an instance with multiple string properties)?